third_party/2and3/requests/utils.pyi

```diff
@@ -41,9 +41,9 @@ def dotted_netmask(mask): ...
 def is_ipv4_address(string_ip): ...
 def is_valid_cidr(string_network): ...
 def set_environ(env_name, value): ...
-def should_bypass_proxies(url): ...
-def get_environ_proxies(url): ...
-def select_proxy(url, proxies): ...
+def should_bypass_proxies(url, no_proxy: Iterable[AnyStr]) -> bool: ...
+def get_environ_proxies(url, no_proxy: Iterable[AnyStr] = ...) -> Dict[Any, Any]: ...
+def select_proxy(url, proxies: Dict[Any, Any]): ...
```

Comment:
There is this line in the implementation: `proxies = proxies or {}`, so `None` is accepted. Also `Mapping` should be sufficient:

```suggestion
def select_proxy(url: Text, proxies: Optional[Mapping[Any, Any]]): ...
```